src/graph_notebook/magics/streams.py

```diff
@@ -0,0 +1,225 @@
+import re
+import json
+import urllib.request
+import urllib.error
+import ipywidgets as widgets
+import queue
+from IPython.display import display, HTML, clear_output
+from graph_notebook.neptune.client import STREAM_AT, STREAM_AFTER, STREAM_TRIM, STREAM_EXCEPTION_NOT_FOUND
+class EventId:
+    def __init__(self, commit_num=1, op_num=1):
+        self.commit_num = int(commit_num)
+        self.op_num = int(op_num)
+        
+    
+    def update(self, event_id):
+        if event_id is not None:
+            self.commit_num = event_id.commit_num
+            self.op_num = event_id.op_num
+            
```

Comment:
Nit picking here but some functions have 2 blank lines after them and some one, we should keep consistent

Reply:
We probably need to run a PEP-8 over all the files in a future update

Reply:
Yeah our integration tests use to enforce style via Flake8 but since they don't work now :(

Reply:
We're still able to run the PEP-8 check, it's in our unit test workflow here:

https://github.com/aws/graph-notebook/blob/262454aec669822b91c4731419ac2f093ba949e4/.github/workflows/unit.yml#L20

The current check probably just isn't strict enough to catch some of the violations mentioned.